### PR TITLE
Social a11y fix

### PR DESCRIPTION
**Pull request by devoeas**
replaced alt text in footer social icons with aria labels to avoid double screen reading

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -109,18 +109,18 @@ <h2 id="getinvolved">Get Involved
...
-				<a href="https://www.facebook.com/profile.php?id=61555332203505" target="_blank"><img
-						src="img/socials/facebook.svg" alt="Facebook" class="social-icon"><img
-						src="img/socials/facebook-hover.svg" alt="Facebook" class="social-icon-hover"></a>
+				<a href="https://x.com/tbfighters" target="_blank" aria-label="Twitter">
+					<img src="img/socials/twitter.svg" class="social-icon">
+					<img src="img/socials/twitter-hover.svg" class="social-icon-hover"></a>
+				<a href="https://www.instagram.com/tbfighters" target="_blank" aria-label="Instagram">
+					<img src="img/socials/insta.svg" class="social-icon">
+					<img src="img/socials/insta-hover.svg" class="social-icon-hover"></a>
+				<a href="https://www.youtube.com/@TB_Fighters" target="_blank" aira-label="YouTube"><img
```
Comment (jwheron) [inline about.html:118]: Typo - aira instead of aria - it looks like this is true for each page.

**Comment by YummyOreo** [comment]
Can you also change it in update-action/footer.html

**Comment by YummyOreo** [comment]
There is also another couple in action.html

File: update-action/footer.html [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-			<img src="{prefix}img/socials/twitter-hover.svg" alt="Twitter" class="social-icon-hover"></a>
-		<a href="https://www.instagram.com/tbfighters" target="_blank">
-			<img src="{prefix}img/socials/insta.svg" alt="Instagram" class="social-icon">
-			<img src="{prefix}img/socials/insta-hover.svg" alt="Instagram" class="social-icon-hover"></a>
-		<a href="https://www.youtube.com/@TB_Fighters" target="_blank"><img
-				src="{prefix}img/socials/youtube.svg" alt="Youtube" class="social-icon"><img
-				src="{prefix}img/socials/youtube-hover.svg" alt="Youtube" class="social-icon-hover"></a>
-		<a href="https://www.facebook.com/profile.php?id=61555332203505" target="_blank"><img
-				src="{prefix}img/socials/facebook.svg" alt="Facebook" class="social-icon"><img
-				src="{prefix}img/socials/facebook-hover.svg" alt="Facebook" class="social-icon-hover"></a>
```
Comment (YummyOreo) [inline update-action/footer.html:14]: The {prefix} part is needed for sub pages like the ones in the template directory

**Pull request closed without merging**